app/src/main/java/com/example/cmput301f18t26/icare/Controllers/DataController.java

```diff
@@ -1,12 +1,15 @@
 package com.example.cmput301f18t26.icare.Controllers;
 
 class DataController {
-    private static final DataController ourInstance = new DataController();
 
-    static DataController getInstance() {
-        return ourInstance;
+    public DataController() {
     }
 
-    private DataController() {
+    public void fetch(){
```

Comment:
fetch and save are fine as is, they dont need to be implemented, adding a comment for each stating what they do would be helpful as these names are fairly abstract without an implementation. 

eg. 

// fetch: this method makes a GET request to ElasticSearch and grabs the latest data pertaining to our user, overwriting our current cached data.

// save: this method makes a POST/Patch request to ElasticSearch to merge our locally cached data changes with what is stored in our hosted database.

Reply:
Thanks Tony very helpful!